Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Core/EBusEventHandler.cpp

```diff
@@ -518,7 +518,7 @@ namespace ScriptCanvas
                 {
                     const AZ::BehaviorParameter& argument(event.m_parameters[AZ::eBehaviorBusForwarderEventIndices::Result]);
                     Data::Type inputType(AZ::BehaviorContextHelper::IsStringParameter(argument) ? Data::Type::String() : Data::FromAZType(argument.m_typeId));
-                    const AZStd::string argName(AZStd::string::format("Result: %s", Data::GetName(inputType).data()).data());
+                    const AZStd::string argName(Data::GetName(inputType).data());
```

Comment:
repeat: `.data()` is not needed
```suggestion
                    const AZStd::string argName(Data::GetName(inputType));
```